warp/paddle.py

```diff
@@ -11,6 +11,7 @@
 from typing import TYPE_CHECKING, Optional, Union
 
 import numpy
+from paddle.base.libpaddle import CPUPlace, CUDAPinnedPlace, CUDAPlace, Place
```

Comment:
Hey @HydrogenSulfate , it looks like this import can't be at the top-level since it assumes the user has installed Paddle. Check out the build errors for more details.

Reply:
Thanks for reply and changed to inline import